src/components/progress-list-card/ASProgressListCard.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from 'react'
+import { View, Text, Image, TouchableOpacity } from 'react-native'
+import * as Progress from 'react-native-progress'
+
+import ASPopUp from '../pop-up/ASPopUp'
+import { ResizeMode, PROGRESS_CARD_IMAGE_SOURCE, nextIcon } from '../../constants'
+import { COLORS, SPACING } from '../../theme'
+import { IProgressProps } from '../../types'
+
+import { styles } from './asProgressListCard-styles'
+
+const ASProgressListCard = (props: IProgressProps) => {
```

Comment:
import and use like this : import { IProgressProps as IASProgressListCardProps} from '../../types'